Update the `compiler-builtins` subtree

tgross35 opened the pull request:
Update the Josh subtree to https://github.com/rust-lang/compiler-builtins/commit/8aba4c899ee8.

r? @ghost

tgross35 committed:
josh-sync: Replace `#xxxx`-style links in messages

Often our short summaries will pick up a Bors "Auto merge of #xxxx ...`
commit message. Replace these with something like `rust-lang/rust#1234`
to avoid broken links when going between repositories.

folkertdev committed:
use `is_multiple_of` to check if an addr is aligned

folkertdev committed:
use `extern "custom"` on naked functions with a custom calling convention

Urgau committed:
Add minimal triagebot config

This PR adds a minimal `triagebot.toml` config to make contributions to
this repository respect upstream rust-lang/rust conventions and avoid
issues when syncing this subtree.

tgross35 committed:
symcheck: Add a wrapper around an archive

Rather than re-opening the archive file for each check, add a wrapper
that keeps the data in memory. Additionally, collect the `--target`
argument so it can be used within this crate.

folkertdev committed:
Use `asm_cfg` in `probestack`

cc https://www.github.com/rust-lang/rust/issues/140364

quaternic committed:
apply suggestions for clippy::manual_is_multiple_of in libm-test

tgross35 committed:
Use the compiler to determine whether or not to enable `f16` and `f128`

Currently we whether or not to build and test `f16` and `f128` support
mostly based on the target triple. This isn't always accurate, however,
since support also varies by backend and the backend version.

Since recently, `rustc` is aware of this with the unstable config option
`target_has_reliable_{f16,f128}`, which better represents when the types
are actually expected to be available and usable. Switch our
compiler-builtins and libm configuration to use this by probing `rustc`
for the target's settings.

A few small `cfg` fixes are needed with this.

quaternic committed:
libm: Improved integer utilities, implement shifts and bug fixes for i256 and u256

`i256` and `u256`
- operators now use the same overflow convention as primitives
- implement `<<` and `-` (previously just `>>` and `+`)
- implement `Ord` correctly (the previous `PartialOrd` was broken)
- correct `i256::SIGNED` to `true`

The `Int`-trait is extended with `trailing_zeros`, `carrying_add`, and
`borrowing_sub`.

tgross35 commented:
@bors r+ rollup=never

bors commented:
:pushpin: Commit be35d37d8b6c206576ff3cdcad956d921b47cc6b has been approved by `tgross35`

It is now in the [queue](https://bors.rust-lang.org/homu/queue/rust) for this repository.

<!-- @bors r=tgross35 be35d37d8b6c206576ff3cdcad956d921b47cc6b -->
<!-- homu: {"type":"Approved","sha":"be35d37d8b6c206576ff3cdcad956d921b47cc6b","approver":"tgross35","queue":"https://bors.rust-lang.org/homu/queue/rust"} -->

bors commented:
:hourglass: Testing commit be35d37d8b6c206576ff3cdcad956d921b47cc6b with merge a6503b16b73b3b101532760a04a4392a51551583...
<!-- homu: {"type":"BuildStarted","head_sha":"be35d37d8b6c206576ff3cdcad956d921b47cc6b","merge_sha":"a6503b16b73b3b101532760a04a4392a51551583"} -->

rust-log-analyzer commented:
The job **`aarch64-apple`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/16074825526/job/45367551998) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/45367551998) [(plain)](https://github.com/rust-lang/rust/commit/a6503b16b73b3b101532760a04a4392a51551583/checks/45367551998/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
---- [run-make] tests/run-make/apple-slow-tls stdout ----

error: rmake recipe failed to complete
status: exit status: 1
command: cd "/Users/runner/work/rust/rust/build/aarch64-apple-darwin/test/run-make/apple-slow-tls/rmake_out" && env -u RUSTFLAGS AR="ar" BUILD_ROOT="/Users/runner/work/rust/rust/build/aarch64-apple-darwin" CARGO="/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage1-tools-bin/cargo" CC="/Applications/Xcode_15.4.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang" CC_DEFAULT_FLAGS="-ffunction-sections -fdata-sections -fPIC --target=arm64-apple-macosx -mmacosx-version-min=11 -isysroot /Applications/Xcode_15.4.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.5.sdk" CXX="/Applications/Xcode_15.4.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++" CXX_DEFAULT_FLAGS="-ffunction-sections -fdata-sections -fPIC --target=arm64-apple-macosx -mmacosx-version-min=11 -isysroot /Applications/Xcode_15.4.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.5.sdk -stdlib=libc++" DYLD_LIBRARY_PATH="/Users/runner/work/rust/rust/build/aarch64-apple-darwin/bootstrap-tools/aarch64-apple-darwin/release/deps:/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage0/lib/rustlib/aarch64-apple-darwin/lib" HOST_RUSTC_DYLIB_PATH="/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage2/lib" LD_LIB_PATH_ENVVAR="DYLD_LIBRARY_PATH" LLVM_BIN_DIR="/Users/runner/work/rust/rust/build/aarch64-apple-darwin/ci-llvm/bin" LLVM_COMPONENTS="aarch64 aarch64asmparser aarch64codegen aarch64desc aarch64disassembler aarch64info aarch64utils aggressiveinstcombine all all-targets amdgpu amdgpuasmparser amdgpucodegen amdgpudesc amdgpudisassembler amdgpuinfo amdgputargetmca amdgpuutils analysis arm armasmparser armcodegen armdesc armdisassembler arminfo armutils asmparser asmprinter avr avrasmparser avrcodegen avrdesc avrdisassembler avrinfo binaryformat bitreader bitstreamreader bitwriter bpf bpfasmparser bpfcodegen bpfdesc bpfdisassembler bpfinfo cfguard cgdata codegen codegentypes core coroutines coverage csky cskyasmparser cskycodegen cskydesc cskydisassembler cskyinfo debuginfobtf debuginfocodeview debuginfodwarf debuginfogsym debuginfologicalview debuginfomsf debuginfopdb demangle dlltooldriver dwarflinker dwarflinkerclassic dwarflinkerparallel dwp engine executionengine extensions filecheck frontendatomic frontenddriver frontendhlsl frontendoffloading frontendopenacc frontendopenmp fuzzercli fuzzmutate globalisel hexagon hexagonasmparser hexagoncodegen hexagondesc hexagondisassembler hexagoninfo hipstdpar instcombine instrumentation interfacestub interpreter ipo irprinter irreader jitlink libdriver lineeditor linker loongarch loongarchasmparser loongarchcodegen loongarchdesc loongarchdisassembler loongarchinfo lto m68k m68kasmparser m68kcodegen m68kdesc m68kdisassembler m68kinfo mc mca mcdisassembler mcjit mcparser mips mipsasmparser mipscodegen mipsdesc mipsdisassembler mipsinfo mirparser msp430 msp430asmparser msp430codegen msp430desc msp430disassembler msp430info native nativecodegen nvptx nvptxcodegen nvptxdesc nvptxinfo objcarcopts objcopy object objectyaml option orcdebugging orcjit orcshared orctargetprocess passes powerpc powerpcasmparser powerpccodegen powerpcdesc powerpcdisassembler powerpcinfo profiledata remarks riscv riscvasmparser riscvcodegen riscvdesc riscvdisassembler riscvinfo riscvtargetmca runtimedyld sandboxir scalaropts selectiondag sparc sparcasmparser sparccodegen sparcdesc sparcdisassembler sparcinfo support symbolize systemz systemzasmparser systemzcodegen systemzdesc systemzdisassembler systemzinfo tablegen target targetparser telemetry textapi textapibinaryreader transformutils vectorize webassembly webassemblyasmparser webassemblycodegen webassemblydesc webassemblydisassembler webassemblyinfo webassemblyutils windowsdriver windowsmanifest x86 x86asmparser x86codegen x86desc x86disassembler x86info x86targetmca xray xtensa xtensaasmparser xtensacodegen xtensadesc xtensadisassembler xtensainfo" LLVM_FILECHECK="/Users/runner/work/rust/rust/build/aarch64-apple-darwin/ci-llvm/bin/FileCheck" NODE="/opt/homebrew/bin/node" PYTHON="/usr/bin/python3" RUSTC="/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage2/bin/rustc" RUSTDOC="/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage2/bin/rustdoc" SOURCE_ROOT="/Users/runner/work/rust/rust" TARGET="aarch64-apple-darwin" TARGET_EXE_DYLIB_PATH="/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage2/lib/rustlib/aarch64-apple-darwin/lib" "/Users/runner/work/rust/rust/build/aarch64-apple-darwin/test/run-make/apple-slow-tls/rmake"
stdout: none
--- stderr -------------------------------
command failed at line 30
Command { cmd: DYLD_LIBRARY_PATH="/Users/runner/work/rust/rust/build/aarch64-apple-darwin/test/run-make/apple-slow-tls/rmake_out:/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage2/lib:/Users/runner/work/rust/rust/build/aarch64-apple-darwin/bootstrap-tools/aarch64-apple-darwin/release/deps:/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage0/lib/rustlib/aarch64-apple-darwin/lib" "/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage1-tools-bin/cargo" "b" "--manifest-path" "tls_test/Cargo.toml" "--target" "t.json" "-Zbuild-std=std,core,panic_abort", stdin_buf: None, stdin: None, stdout: None, stderr: None, drop_bomb: DropBomb { command: "/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage1-tools-bin/cargo", defused: true, armed_location: Location { file: "src/tools/run-make-support/src/external_deps/cargo.rs", line: 8, column: 19 } }, already_executed: true }
output status: `exit status: 101`
=== STDOUT ===



=== STDERR ===
   Compiling compiler_builtins v0.1.160 (/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage2/lib/rustlib/src/rust/library/compiler-builtins/compiler-builtins)
---
    = help: run `rustc --print target-list` for a list of built-in targets


  thread 'main' panicked at /Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage2/lib/rustlib/src/rust/library/compiler-builtins/compiler-builtins/configure.rs:48:9:
  failed to run `RUSTC_BOOTSTRAP="1" "/Users/runner/work/rust/rust/build/aarch64-apple-darwin/stage2/bin/rustc" "--print=cfg" "--target" "t"`
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
------------------------------------------


```

</details>

bors commented:
:broken_heart: Test failed - [checks-actions](https://github.com/rust-lang/rust/actions/runs/16074825526/job/45371554148)
<!-- homu: {"type":"BuildFailed","builder_url":"https://github.com/rust-lang/rust/actions/runs/16074825526/job/45371554148","builder_name":"checks-actions"} -->

tgross35 committed:
Remove the `let_chains` feature now that it is stable

tgross35 committed:
symcheck: Make `target` a positional argument

This makes it more obvious what we intend to check rather than looking
for `--target`.

tgross35 committed:
symcheck: Improve diagnostics from spawned Cargo

Rather than printing the entire JSON dump, use the rendered version.

tgross35 committed:
Remove unused custom target JSON files

8521530f4938 ("Fix __divsi3 and __udivsi3 on thumbv6m targets") removed
tests that use these `thumb*-linux` target files in favor of tests that
use the `thumb*-none` targets, which are available via Rustup. The JSON
files haven't been used since then and are outdated, so remove them.

tgross35 committed:
Test building custom targets and resolve an issue probing `rustc`

The `rustc` probe done in our build scripts needs to pass `--target` to
get the correct configuration, which usually comes from the `TARGET`
environment variable. However, for targets specified via a `target.json`
file, `TARGET` gets set to the file name without an extension or path.
`rustc` will check a search path to attempt to locate the file, but this
is likely to fail since the directory where Cargo invokes build scripts
(and hence where those scripts invoke `rustc`) might not have any
relation to the JSON spec file.

Resolve this for now by leaving `f16` and `f128` disabled if the `rustc`
command fails. Result of the discussion at CARGO-14208 may eventually
provide a better solution.

A CI test is also added since custom JSON files are an edge case that
could fail in other ways. I verified this fails without the fix here.
The JSON file is the output for `thumbv7em-none-eabi`, just renamed so
`rustc` doesn't identify it.

bors commented:
:pushpin: Commit 016bc61312b85fae0f0f53c56946239b712b4604 has been approved by `tgross35`

It is now in the [queue](https://bors.rust-lang.org/homu/queue/rust) for this repository.

<!-- @bors r=tgross35 016bc61312b85fae0f0f53c56946239b712b4604 -->
<!-- homu: {"type":"Approved","sha":"016bc61312b85fae0f0f53c56946239b712b4604","approver":"tgross35","queue":"https://bors.rust-lang.org/homu/queue/rust"} -->

bors commented:
:hourglass: Testing commit 016bc61312b85fae0f0f53c56946239b712b4604 with merge 4b909fc1e6206b9e625d3d6507998a2a99488f24...
<!-- homu: {"type":"BuildStarted","head_sha":"016bc61312b85fae0f0f53c56946239b712b4604","merge_sha":"4b909fc1e6206b9e625d3d6507998a2a99488f24"} -->

rust-log-analyzer commented:
The job **`x86_64-mingw-1`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/16084621066/job/45394187297) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/45394187297) [(plain)](https://github.com/rust-lang/rust/commit/4b909fc1e6206b9e625d3d6507998a2a99488f24/checks/45394187297/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
---- [ui] tests\ui\float\conv-bits-runtime-const.rs stdout ----

error: test run failed!
status: exit code: 101
command: PATH="D:\a\rust\rust\build\x86_64-pc-windows-gnu\stage2\lib\rustlib\x86_64-pc-windows-gnu\lib;D:\a\rust\rust\build\x86_64-pc-windows-gnu\bootstrap-tools\x86_64-pc-windows-gnu\release\deps;D:\a\rust\rust\ninja;D:\a\rust\rust\mingw64\bin;C:\msys64\usr\bin;D:\a\rust\rust\sccache;C:\Program Files\MongoDB\Server\5.0\bin;C:\aliyun-cli;C:\vcpkg;C:\Program Files (x86)\NSIS;C:\tools\zstd;C:\Program Files\Mercurial;C:\hostedtoolcache\windows\stack\3.7.1\x64;C:\cabal\bin;C:\ghcup\bin;C:\mingw64\bin;C:\Program Files\dotnet;C:\Program Files\MySQL\MySQL Server 8.0\bin;C:\Program Files\R\R-4.4.2\bin\x64;C:\SeleniumWebDrivers\GeckoDriver;C:\SeleniumWebDrivers\EdgeDriver;C:\SeleniumWebDrivers\ChromeDriver;C:\Program Files (x86)\sbt\bin;C:\Program Files (x86)\GitHub CLI;C:\Program Files\Git\bin;C:\Program Files (x86)\pipx_bin;C:\npm\prefix;C:\hostedtoolcache\windows\go\1.24.4\x64\bin;C:\hostedtoolcache\windows\Python\3.9.13\x64\Scripts;C:\hostedtoolcache\windows\Python\3.9.13\x64;C:\hostedtoolcache\windows\Ruby\3.3.8\x64\bin;C:\Program Files\OpenSSL\bin;C:\tools\kotlinc\bin;C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\8.0.452-9\x64\bin;C:\Program Files\ImageMagick-7.1.1-Q16-HDRI;C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin;C:\ProgramData\kind;C:\ProgramData\Chocolatey\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0;C:\Windows\System32\OpenSSH;C:\Program Files\dotnet;C:\Program Files\PowerShell\7;C:\Program Files\Microsoft\Web Platform Installer;C:\Program Files\TortoiseSVN\bin;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn;C:\Program Files\Microsoft SQL Server\150\Tools\Binn;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit;C:\Program Files (x86)\WiX Toolset v3.14\bin;C:\Program Files\Microsoft SQL Server\130\DTS\Binn;C:\Program Files\Microsoft SQL Server\140\DTS\Binn;C:\Program Files\Microsoft SQL Server\150\DTS\Binn;C:\Program Files\Microsoft SQL Server\160\DTS\Binn;C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\perl\bin;C:\ProgramData\chocolatey\lib\pulumi\tools\Pulumi\bin;C:\Program Files\CMake\bin;C:\ProgramData\chocolatey\lib\maven\apache-maven-3.9.10\bin;C:\Program Files\Microsoft Service Fabric\bin\Fabric\Fabric.Code;C:\Program Files\Microsoft SDKs\Service Fabric\Tools\ServiceFabricLocalClusterManager;C:\Program Files\nodejs;C:\Program Files\Git\cmd;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Program Files\GitHub CLI;C:\tools\php;C:\Program Files (x86)\sbt\bin;C:\Program Files\Amazon\AWSCLIV2;C:\Program Files\Amazon\SessionManagerPlugin\bin;C:\Program Files\Amazon\AWSSAMCLI\bin;C:\Program Files\Microsoft SQL Server\130\Tools\Binn;C:\Program Files\LLVM\bin;C:\Program Files (x86)\LLVM\bin;C:\Users\runneradmin\.dotnet\tools;C:\Users\runneradmin\.cargo\bin;C:\Users\runneradmin\AppData\Local\Microsoft\WindowsApps" "D:\\a\\rust\\rust\\build\\x86_64-pc-windows-gnu\\test\\ui\\float\\conv-bits-runtime-const\\a.exe"
stdout: none
--- stderr -------------------------------

thread 'main' panicked at D:\a\rust\rust\tests\ui\float\conv-bits-runtime-const.rs:43:5:
assertion `left == right` failed
---
test result: FAILED. 19032 passed; 1 failed; 289 ignored; 0 measured; 21 filtered out; finished in 1110.02s

Some tests failed in compiletest suite=ui mode=ui host=x86_64-pc-windows-gnu target=x86_64-pc-windows-gnu
Build completed unsuccessfully in 1:36:50
make: *** [Makefile:123: ci-mingw-x] Error 1
  local time: Sat Jul  5 06:28:00 CUT 2025
  network time: Sat, 05 Jul 2025 06:28:00 GMT
##[error]Process completed with exit code 2.
Post job cleanup.
[command]"C:\Program Files\Git\bin\git.exe" version
```

</details>

bors commented:
:broken_heart: Test failed - [checks-actions](https://github.com/rust-lang/rust/actions/runs/16084621066/job/45395886674)
<!-- homu: {"type":"BuildFailed","builder_url":"https://github.com/rust-lang/rust/actions/runs/16084621066/job/45395886674","builder_name":"checks-actions"} -->

tgross35 commented:
@bors2 try jobs=x86_64-mingw-*,aarch64-apple

tgross35 commented:
@bors r+

bors commented:
:pushpin: Commit 016bc61312b85fae0f0f53c56946239b712b4604 has been approved by `tgross35`

It is now in the [queue](https://bors.rust-lang.org/homu/queue/rust) for this repository.

<!-- @bors r=tgross35 016bc61312b85fae0f0f53c56946239b712b4604 -->
<!-- homu: {"type":"Approved","sha":"016bc61312b85fae0f0f53c56946239b712b4604","approver":"tgross35","queue":"https://bors.rust-lang.org/homu/queue/rust"} -->

bors commented:
:hourglass: Testing commit 016bc61312b85fae0f0f53c56946239b712b4604 with merge fd2e8b91494f1d8da2419b6583d5f4ec4e72f24b...
<!-- homu: {"type":"BuildStarted","head_sha":"016bc61312b85fae0f0f53c56946239b712b4604","merge_sha":"fd2e8b91494f1d8da2419b6583d5f4ec4e72f24b"} -->

rust-log-analyzer commented:
The job **`dist-x86_64-msvc`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/16171476725/job/45645978422) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/45645978422) [(plain)](https://github.com/rust-lang/rust/commit/fd2e8b91494f1d8da2419b6583d5f4ec4e72f24b/checks/45645978422/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
[2025-07-09T14:47:55Z DEBUG collector::compile::benchmark] Benchmark iteration 1/1
[2025-07-09T14:47:55Z INFO  collector::compile::execute] run_rustc with incremental=false, profile=Check, scenario=Some(Full), patch=None, backend=Llvm, target=X86_64UnknownLinuxGnu, phase=benchmark
[2025-07-09T14:47:55Z DEBUG collector::compile::execute] "\\\\?\\C:\\a\\rust\\rust\\build\\x86_64-pc-windows-msvc\\stage0\\bin\\cargo.exe" "rustc" "--manifest-path" "Cargo.toml" "-p" "path+file:///C:/Users/RUNNER~1/AppData/Local/Temp/.tmpgTtMxH#bitmaps@3.2.1" "--profile" "check" "--" "--wrap-rustc-with" "Eprintln"
[2025-07-09T14:47:56Z INFO  collector::compile::execute] run_rustc with incremental=true, profile=Check, scenario=Some(IncrFull), patch=None, backend=Llvm, target=X86_64UnknownLinuxGnu, phase=benchmark
[2025-07-09T14:47:56Z DEBUG collector::compile::execute] "\\\\?\\C:\\a\\rust\\rust\\build\\x86_64-pc-windows-msvc\\stage0\\bin\\cargo.exe" "rustc" "--manifest-path" "Cargo.toml" "-p" "path+file:///C:/Users/RUNNER~1/AppData/Local/Temp/.tmpgTtMxH#bitmaps@3.2.1" "--profile" "check" "--" "--wrap-rustc-with" "Eprintln" "-C" "incremental=C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\.tmpgTtMxH\\incremental-state"
[2025-07-09T14:47:57Z INFO  collector::compile::execute] run_rustc with incremental=true, profile=Check, scenario=Some(IncrUnchanged), patch=None, backend=Llvm, target=X86_64UnknownLinuxGnu, phase=benchmark
[2025-07-09T14:47:57Z DEBUG collector::compile::execute] "\\\\?\\C:\\a\\rust\\rust\\build\\x86_64-pc-windows-msvc\\stage0\\bin\\cargo.exe" "rustc" "--manifest-path" "Cargo.toml" "-p" "path+file:///C:/Users/RUNNER~1/AppData/Local/Temp/.tmpgTtMxH#bitmaps@3.2.1" "--profile" "check" "--" "--wrap-rustc-with" "Eprintln" "-C" "incremental=C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\.tmpgTtMxH\\incremental-state"
[2025-07-09T14:47:58Z DEBUG collector::compile::benchmark] applying patch println
[2025-07-09T14:47:58Z DEBUG collector::compile::benchmark::patch] applying println to "C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\.tmpgTtMxH"
[2025-07-09T14:47:58Z INFO  collector::compile::execute] run_rustc with incremental=true, profile=Check, scenario=Some(IncrPatched), patch=Some(Patch { index: 0, name: PatchName("println"), path: "0-println.patch" }), backend=Llvm, target=X86_64UnknownLinuxGnu, phase=benchmark
[2025-07-09T14:47:58Z DEBUG collector::compile::execute] "\\\\?\\C:\\a\\rust\\rust\\build\\x86_64-pc-windows-msvc\\stage0\\bin\\cargo.exe" "rustc" "--manifest-path" "Cargo.toml" "-p" "path+file:///C:/Users/RUNNER~1/AppData/Local/Temp/.tmpgTtMxH#bitmaps@3.2.1" "--profile" "check" "--" "--wrap-rustc-with" "Eprintln" "-C" "incremental=C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\.tmpgTtMxH\\incremental-state"
Running bitmaps-3.2.1: Debug + [Full, IncrFull, IncrUnchanged, IncrPatched] + Llvm + X86_64UnknownLinuxGnu
[2025-07-09T14:47:59Z DEBUG collector::compile::benchmark] Benchmark iteration 1/1
[2025-07-09T14:47:59Z INFO  collector::compile::execute] run_rustc with incremental=false, profile=Debug, scenario=Some(Full), patch=None, backend=Llvm, target=X86_64UnknownLinuxGnu, phase=benchmark
[2025-07-09T14:47:59Z DEBUG collector::compile::execute] "\\\\?\\C:\\a\\rust\\rust\\build\\x86_64-pc-windows-msvc\\stage0\\bin\\cargo.exe" "rustc" "--manifest-path" "Cargo.toml" "-p" "path+file:///C:/Users/RUNNER~1/AppData/Local/Temp/.tmpF6eo83#bitmaps@3.2.1" "--" "--wrap-rustc-with" "Eprintln"
[2025-07-09T14:48:00Z INFO  collector::compile::execute] run_rustc with incremental=true, profile=Debug, scenario=Some(IncrFull), patch=None, backend=Llvm, target=X86_64UnknownLinuxGnu, phase=benchmark
```

</details>

bors commented:
:broken_heart: Test failed - [checks-actions](https://github.com/rust-lang/rust/actions/runs/16171476725/job/45657970139)
<!-- homu: {"type":"BuildFailed","builder_url":"https://github.com/rust-lang/rust/actions/runs/16171476725/job/45657970139","builder_name":"checks-actions"} -->

tgross35 commented:
@bors2 try jobs=dist-x86_64-msvc

tgross35 commented:
Spurious?
@bors r+

bors commented:
:hourglass: Testing commit 016bc61312b85fae0f0f53c56946239b712b4604 with merge 607fbd8d3e25997737a0bf25b215e581b922f1b0...
<!-- homu: {"type":"BuildStarted","head_sha":"016bc61312b85fae0f0f53c56946239b712b4604","merge_sha":"607fbd8d3e25997737a0bf25b215e581b922f1b0"} -->
